client/src/components/elements/Login.tsx

```diff
@@ -0,0 +1,14 @@
+import Image from "../../assets/clock.svg";
+import {Link} from "react-router-dom";
+
+const Login = () => {
+    return (
+        <Link className="clickable" to="/login">
+            <div className="applet">
+                <img className="image" src={Image} alt={"Login icon"}/>
+            </div>
+        </Link>
+    );
+};
+
+export default Login;
```

Comment:
@JingtingL should prob add newlines in the end of the files cuz of [this](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3)